### PR TITLE
Added kafka-support related changes

### PR DESCRIPTION
**Pull request by vaibhavatul47**
- Updated docker-compose with kafka related ports.
- Added new sample for kafka produce-consume

### CODE REVIEW

File: README.md
Comment (SayanMS09) [inline README.md:98]: Can we use this exact string (with the "UseDevelopmentEmulator=true") at the end in our .net sample code too?

**Review by SayanMS09 — CHANGES_REQUESTED** [review]
Use same connection string which is provided in README and in sample code.

**Comment by vaibhavatul47** [comment]
@microsoft-github-policy-service agree [company="microsoft"]

**Review by SayanMS09 — APPROVED** [review]
Approved. Lets wait for Sannidhya's Approval and right time to merge this change.

**Review by Saglodha — APPROVED** [review]

**Pull request merged**
